### ISSUE TITLE
imagePullSecret support?

### ISSUE THREAD

**Issue by waldner** [skupperproject/skupper#853]
When using images from private repositories (eg with the varions `SKUPPER_*_IMAGE` environment variables), I can't find a way to specify an imagePullSecret, either for the pods or for the service accounts.
Is this supported?

**Comment by grs** [comment]
No, there is no support for this yet. (You could patch or even precreate the service accounts as a workaround until there is such support built in).

**Comment by waldner** [comment]
Ok thanks.
